internal/pkg/manifest/validate.go

```diff
@@ -810,7 +874,16 @@ func (d *DeadLetterQueue) Validate() error {
 }
 
 // Validate returns nil if OverrideRule is configured correctly.
-func (*OverrideRule) Validate() error {
+func (r *OverrideRule) Validate() error {
+	if r == nil {
+		return nil
+	}
+	for _, s := range invalidTaskDefOverridePathRegexp {
+		re := regexp.MustCompile(fmt.Sprintf(`^%s$`, s))
+		if re.MatchString(r.Path) {
+			return fmt.Errorf("%s cannot be overidden with a custom value", s)
```

Comment:
What do you think about
```suggestion
			return fmt.Errorf(`"%s" cannot be overridden with a custom value`, s)
```